summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -230,10 +230,47 @@ gitHubFiles Settings{..} =
           - stack test --system-ghc
         |]
 
-
-    -- create appveyor.yml template
     appVeyorYml :: Text
     appVeyorYml =
+        if settingsCabal
+        then appVeyorYmlCabal
+        else appVeyorYmlStack
```

Comment:
Probably later we would figure out how to use both, but this is out of the scope of this issue.

Reply:
I agree. Also, according to my experience, `cabal` in AppVeyor runs faster with these two settings compared. And it works without any problems on the package that we generate. So I prefer to keep `cabal` the default unless you explicitly tell that you don't want to support `cabal`.

Probably `stack` also can work faster, but I'm not using AppVeyor much and don't have much experience with Windows package managers, so can't improve things a lot on my own here. Maybe later somebody will write `Dead simple AppVeyor settings for both Cabal and Stack` 🙂